### DISCUSSION TITLE
Parking after job cancelled and BLtouch HS behaviour

### DISCUSSION THREAD

**Discussion by breaktech** [Jyers/Marlin#429] [Ideas]
Congratulation for the firmware, really well made, I like it very much.
There are just two things that I liked most in my previous firmware, it could be nice to have them imnplemented here.

1) Parking position after job cancel: in my previous firmware (smith 3d) the bed cameall the way out on the outside leaving the machine ready to be cleaned and the steppers were released so that cleaning is more easy.

2) High Speed BLtouch: the probe is shot off while the gantry is still low and every time it knock hard on the bed, this will lead to a premature dead of it I suppose... I understand that this is the behaviour needed for high speed but if the probe is taken out from its parking just a second later it won't knock the bed and high speed would still be possible...

Anybody here share my thoughts? Am I wrong somehow?

**Comment by cruzer619** [comment]
1.) you can do all that in your slicer.. on your ending gcode..  This is my cura ending gcode.. 

G0 X0 Y180 F5000 ; Move Heat Bed to the front for easy print removal

EDIT:  just realized you said cancel... well I use Octoprint.. in my GCode script there I do the same.. when I cancel a print job moves the bed as well same setting.. so I can easily remove the bad print.. and print again.

2.) not sure about that.. as my nozzle is never that close to bed after a print.. as in my ending gcode.. I raise it too..

G0 F5000 Z20 ; Move Z Axis up 20mm to allow filament ooze freely

**Reply by breaktech** [reply]
I actually ment the probe being released in a bed levelling process

**Reply by cruzer619** [reply]
right..  since my nozzle is not so close to the bed.. as I move it up after a print job or cancel a print job.. the probe doesn't hit the bed on initialization of the bed level process.
